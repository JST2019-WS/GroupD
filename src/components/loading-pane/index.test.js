/* eslint-env node, jest */

import LoadingPane from "./index";
import { shallow } from 'enzyme'

describe('loading-pane', () => {
    it('should render as expected', () => {
        const component = shallow(<LoadingPane/>);
        expect(component.html()).toMatchSnapshot();
    })
});
