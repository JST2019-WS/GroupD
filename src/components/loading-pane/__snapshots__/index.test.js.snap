// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading-pane should render as expected 1`] = `"<div class=\\"dialog-pane-background\\"><div class=\\"dialog-pane-dialog\\" style=\\"background-color: transparent;\\"><div class=\\"spinner-container\\"><div class=\\"spinner\\"><div></div><div></div><div></div></div></div></div></div>"`;
